.github/workflows/merge_pull_request.yml

```diff
@@ -0,0 +1,20 @@
+name: Merge pull request
+
+on:
+  repository_dispatch:
+    types: [merge-pr]
+
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Merge pull request for ${{ github.event.client_payload.ReleaseBranchName }}
+      uses: juliangruber/merge-pull-request-action@v1
```

Comment:
same as above, you can probably just use octokit for this.

Reply:
Done.